rclpy/rclpy/client.py

```diff
@@ -14,6 +14,7 @@
 
 import threading
 
+from rclpy.graph_listener import GraphEventSubscription as _GraphEventSubscription
```

Comment:
Old habit. It's to stop someone from importing `rclpy.client`, using `rclpy.client.GraphEventSubscription` and having their code break when the implementation changes. I can remove it if you'd like.

Reply:
I would lean towards not using the underscore prefix. If a module wants to be explicit it can define `__all__` (which don't do anywhere at the moment but which would also cover all the other symbols which shouldn't be exposed like `threading`).

Reply:
Removed underscores 
7843c5c

Reply:
Removed underscore for real this time 149ad4312c41729dc83a46f6ff512feaaec8c41d